requirements.txt

```diff
@@ -1,5 +1,6 @@
-numpy==1.13        # rq.filter: ==1.13
-pandas==0.20      # rq.filter: ==0.20
-scipy==0.19         # rq.filter: ==0.19
-statsmodels==0.8    # rq.filter: ==0.8
-pytest==3.1         # rq.filter: =3.1
+numpy==1.14         # rq.filter: ==1.14
```

Comment:
Do we need a requirements.txt?

Reply:
it is used in .appveyor.yaml, but I agree that we should completely switch to conda in the future. #155

Reply:
Actually it is not. I updated the file to make tests work. But, let's postpone it.